scraper/scraper.py

```diff
@@ -0,0 +1,65 @@
+from configparser import ConfigParser
+from sprinters_driver import SprintersDriver
+from scraper_utils import get_random_number_from_range
+import time
+from scraper_utils import get_flat_ox_data, get_flat_otd_data, dump_csv_dict_data
+
+def main():
+    configparser = ConfigParser()
+    configparser.read('config.ini')
```

Comment:
what do u think about moving this config to main config file? (settings.cfg in the main directory)
@odpaleniek1337 @230443

Reply:
it depends for me its cool, but we need to think if all the components would be packed onto one machine & directory
if so, we could do that, otherwise i think its good if all components have their own configs

Reply:
I would prefer it to be consistent for all modules. So:
1. Create separate configs for all modules
2. One global config.

I think the second option is good enough even if we run it separately since we can pull the repository and configure only required part on scraping machine. And its easier to manage when using on one machine.

However, I think its good that we have separate `requirements.txt` and it would be good if ml and web also had it separated, so we do not install unnecessary stuff.

Reply:
requirements are updated in  **scraper** folder

okay, im moving **scraper** config onto the main directory config

Reply:
done in https://github.com/230443/Sprinters/pull/15/commits/05dbe6ae6ebf1a0a9a46985df1d9f065dfbf703a